source/library/HW/Middleware.hs

```diff
@@ -201,25 +201,25 @@ contentSecurityPolicy maybeListmonk =
       "style-src 'self'"
     ]
 
--- | The value of the @Feature-Policy@ header.
--- <https://scotthelme.co.uk/a-new-security-header-feature-policy/>
-featurePolicy :: Text.Text
-featurePolicy =
+-- | The value of the @Permissions-Policy@ header.
+-- <https://scotthelme.co.uk/goodbye-feature-policy-and-hello-permissions-policy//>
```

Comment:
Clickable: https://scotthelme.co.uk/goodbye-feature-policy-and-hello-permissions-policy/